Example/Tests/ExternallyManagedSessionTests.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  ExternallyManagedSessionTests.swift
+//  BitmovinConvivaAnalytics_Tests
+//
+//  Created by Bitmovin on 11.10.18.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import Quick
+import Nimble
+import BitmovinPlayer
+import BitmovinConvivaAnalytics
+import ConvivaSDK
+
+class ExternallyManagedSessionSpec: QuickSpec {
+    // swiftlint:disable:next function_body_length
+    override func spec() {
+        var playerDouble: BitmovinPlayerTestDouble!
+
+        beforeEach {
+            playerDouble = BitmovinPlayerTestDouble()
+            TestHelper.shared.spyTracker.reset()
+            TestHelper.shared.mockTracker.reset()
+        }
+
+        context("Externally Managed Session") {
+            var convivaAnalytics: ConvivaAnalytics!
+            beforeEach {
+                do {
+                    convivaAnalytics = try ConvivaAnalytics(player: playerDouble, customerKey: "")
+                } catch {
+                    fail("ConvivaAnalytics failed with error: \(error)")
+                }
+            }
+
+            afterEach {
+                if convivaAnalytics != nil {
+                    convivaAnalytics = nil
+                }
+            }
+
+            context("initialize session") {
+                var spy: Spy!
+                beforeEach {
+                    spy = Spy(aClass: CISClientTestDouble.self, functionName: "createSession")
+                }
+                context("without source loaded") {
+                    beforeEach {
+                        let playerConfig = PlayerConfiguration()
+                        _ = TestDouble(aClass: playerDouble, name: "config", return: playerConfig)
+                    }
+
+                    it("with asset name provided") {
+                        try? convivaAnalytics.initializeSession(assetName: "MyAsset")
+                        expect(spy).to(haveBeenCalled(withArgs: ["assetName": "MyAsset"]))
+                    }
+
+                    it("throw error without source and asset name") {
```

Comment:
This test will break ability to run tests on a real device. Should we check running destination with a macro?